notebooks/WFC3/persistence/wfc3_ir_persistence.ipynb

```diff
@@ -58,9 +58,9 @@
     "\n",
```

Comment:
<p>Thanks again for making the style updates. I would suggest adding exception handling. Also, we can extract the filename as follows:</p><p> </p><pre class="ql-syntax" spellcheck="false">for im in science_files:
&nbsp; &nbsp; filename = os.path.basename(im)
&nbsp; &nbsp; new_path = os.path.join('.',filename)
&nbsp; &nbsp; os.rename(im, new_path)
&nbsp; &nbsp;&nbsp;
data_directory = './data'

try:
&nbsp; &nbsp; if os.path.isdir(data_directory):
&nbsp; &nbsp; &nbsp; &nbsp; shutil.rmtree(data_directory)
except Exception as e:
&nbsp; &nbsp; print(f"An error occured while deleting the directory {data_directory}: {e}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>